samples/routing/fair-distribution/sample.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Fair load distribution
+summary: Implementing fair load distribution for heterogeneous scaled-out endpoints
+component: Core
+reviewed: 2016-06-21
+tags:
+- Routing
+- Distribution
+- DistributionStrategy
+---
+
+
```

Comment:
Could we have a short intro as in http://docs.particular.net/samples/routing/custom/ to explain what it does on a high level and why would anybody use it?